exercises/changing_text.md

```diff
@@ -0,0 +1,40 @@
+## Changing text
+
+This exercise introduces the **Html blocks** you can use to find and update the text of html elements.
+
+Let's start with the html below in the html area. It represents a sentence with gaps, taken from the popular [Mad Libs](https://en.wikipedia.org/wiki/Mad_Libs) game
+
+An example of a mad lib sentence would be the following:
+
+>  The dog saw the ___ cat
+
+
+Check out the text area, where you should see the following html:
+
+```html
+<p>The <u id="noun1">dog</u> <u id="verb">saw</u> the <u id="adjective">___</u> <u id="noun2">cat</u></p>
+```
+
+This html represents the mad libs sentence we first saw above. Click the "run" button to see the html rendered in the web page. 
+
+<br>
+
+Let's update the contents of the adjective `<u>` element to contain an adjective to describe the cat.
+
+1.  Go to the html blocks and from there add an `at the start` block
+      
+2.  Inside this block, add a `find the element with id` block using the css selector for the id of the adjective `<u>` element
+      
+3.  Inside this block, add a `set the text content to` block 
+
+4. Set the value of the text in this element to pink ( or any other colour you like )
```

Comment:
I think using a color adjectives might make it confusing with the use of color attributes in other exercises. Maybe suggest words like "big, small, scary, angry"?

Reply:
yeah good point